giskard/core/core.py

```diff
@@ -40,7 +40,7 @@ def create_test_function_id(func):
         from giskard.ml_worker.testing.registry.registry import plugins_root
 
         # is_relative_to is only available from python 3.9
-        is_relative = Path(inspect.getfile(func)).relative_to(plugins_root)
+        is_relative = Path(inspect.getfile(func)).is_relative_to(plugins_root)
     except ValueError:
```

Comment:
PS: I guess we can remove the except since it was die to using `remative_to`: https://docs.python.org/3/library/pathlib.html

Reply:
good point